state_management/go/http/README.md

```diff
@@ -1,16 +1,50 @@
 # Dapr state management (HTTP Client)
 
-In this quickstart, you'll create a microservice to demonstrate Dapr's state management API. The service generates messages to store data in a state store.
+In this quickstart, you'll create a microservice to demonstrate Dapr's state management API. The service generates messages to store in a state store. See [Why state management](https://docs.dapr.io/developing-applications/building-blocks/state-management/) to understand when to use this API.
 
-Visit the Dapr documentation on [State Management](https://docs.dapr.io/developing-applications/building-blocks/state-management/) for more information.
+Visit [this](https://docs.dapr.io/developing-applications/building-blocks/state-management/) link for more information about Dapr and State Management.
 
-> **Note:** This example leverages plain HTTP. You can find an example using the Dapr Client SDK (recommended) in the [`sdk` folder](../sdk/).
+> **Note:** This example leverages HTTP `requests` only.  If you are looking for the example using the Dapr Client SDK (recommended) [click here](../sdk/).
 
-This quickstart includes one service: Go client service `order-processor` 
+This quickstart includes one service: Go client service `order-processor`
 
-### Run Go service with Dapr
+## Run all apps with multi-app run template file
 
-1. Run the Go service app with Dapr in the `order-processor` folder:
+This section shows how to run applications at once using [multi-app run template files](https://docs.dapr.io/developing-applications/local-development/multi-app-dapr-run/multi-app-overview/) with `dapr run -f .`.  This enables to you test the interactions between multiple applications.
+
+Open a new terminal window and run  `order-processor` using the multi app run template defined in [dapr.yaml](./dapr.yaml):
+
+1. Run the Go service app with Dapr:
+
+<!-- STEP
+name: Run order-processor service
+expected_stdout_lines:
+  - '== APP - order-processor == Retrieved Order: "{\"orderId\":1}"'
```

Comment:
i'll have to test but I'm surprised this is so different that input we show below

Reply:
The correct format is refreshed in the http output.